client-spark/spark-3/src/main/java/org/apache/spark/shuffle/celeborn/SparkShuffleManager.java

```diff
@@ -116,8 +116,7 @@ private void initializeLifecycleManager() {
                   lifecycleManager.getHost(),
                   lifecycleManager.getPort(),
                   celebornConf,
-                  lifecycleManager.getUserIdentifier(),
-                  true);
+                  lifecycleManager.getUserIdentifier());
```

Comment:
We don't need to initialize the `shuffleClient` instance at the driver side anymore.

Reply:
Yea, you are right